exist-core/src/main/java/org/exist/xquery/pragmas/NoIndexPragma.java

```diff
@@ -30,11 +30,11 @@
 import org.exist.xquery.value.Sequence;
 
 public class NoIndexPragma extends Pragma {
+    public final static String LOCAL_NAME = "no-index";
```

Comment:
See my comment above about why renaming this to `LOCAL_NAME` is problematic and needs to be reversed.